<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-12T17:30:27 IST" name="Default suite" finished-at="2024-03-12T17:30:42 IST" duration-ms="15220">
    <groups>
    </groups>
    <test started-at="2024-03-12T17:30:27 IST" name="Default test" finished-at="2024-03-12T17:30:42 IST" duration-ms="15220">
      <class name="BuydirectTest.TestClass.TC_EnrollPageTest">
        <test-method is-config="true" signature="Loadingpage()[pri:0, instance:BuydirectTest.TestClass.TC_EnrollPageTest@c730b35]" started-at="2024-03-12T17:30:27 IST" name="Loadingpage" finished-at="2024-03-12T17:30:34 IST" duration-ms="6815" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Loadingpage -->
        <test-method signature="TC3_verify_Submit_button()[pri:0, instance:BuydirectTest.TestClass.TC_EnrollPageTest@c730b35]" started-at="2024-03-12T17:30:34 IST" name="TC3_verify_Submit_button" finished-at="2024-03-12T17:30:41 IST" duration-ms="7535" status="FAIL">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 1 out of bounds for length 1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
at BuydirectTest.TestClass.TC_EnrollPageTest.TC3_verify_Submit_button(TC_EnrollPageTest.java:88)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC3_verify_Submit_button -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:BuydirectTest.TestClass.TC_EnrollPageTest@c730b35]" started-at="2024-03-12T17:30:41 IST" name="tearDown" finished-at="2024-03-12T17:30:42 IST" duration-ms="788" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- BuydirectTest.TestClass.TC_EnrollPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
